### ISSUE TITLE
Clarify icon use for off site and different design

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-website#266]
Icons to indicate links go somewhere else are optional. Use them when users are likely to be surprised that the design is different.

* For external links off the W3C website, use the [external-link icon](https://wai-website-theme.netlify.app/components/icons/#external-link)

* For links to pages on the W3C site with different wrapper &mdash; e.g., on a WAI website page, links to /TR/ &mdash; use the [different-view icon](https://wai-website-theme.netlify.app/components/icons/#different-view)

**Comment by shawna-slh** [comment]
@hidde Is this your understanding of which icons to use for what?

(CC: @nitedog )

**Comment by hidde** [comment]
I fear not many users will be familiar with the 'different view' icon, which is why I used the external links icon in the supporting documents templates, I feel we should go ahead with that.

**Comment by shawna-slh** [comment]
I agree that people are familiar with the "links to external site" icon. These links do *not* go to an external site, and thus should not use the external site icon. 

These links stay on the W3C site, just with different nav, etc. That's why we have a different icon for that.  I think we started using these different icons in the 2005 redesign, and haven't gotten any feedback (which doesn't mean it's good, just not bad enough for anyone to take the time to comment :-)

I'm not saying it's a perfect solution. Yet, I do think it's not worth taking the time to explore options right now. I think good to go with previously defined icons.

However, if you think it's worth our time to work more on it, I am very happy to talk through it with you over the phone. Just let me know.

**Comment by hidde** [comment]
To me that icon means “external” more than “external site”. In my mental model, the supporting docs are external to the main WAI website, and going from there to ”All WCAG Guidance” means leaving the, say, Techniques page.

My preference would be to use the external icon, but at least get rid of this “internal” icon as. Maybe we can resolve to having no icon at all?
